src/js/utils/RouterUtil.js

```diff
@@ -186,6 +186,24 @@ const RouterUtil = {
       const blob = new Blob([data], { type: "application/json" });
       global.navigator.msSaveOrOpenBlob(blob, filename);
     }
+  },
+
+  /**
+   * Compair two urls if they have the same host name
+   *
+   * @param {String} urlA The first url
+   * @param {String} urlB The second url
+   * @returns {Boolean} - True if they have the same host, false if they don't
+   */
+  hasSameHost(urlA, urlB) {
+    const parsedUrlA = Util.parseUrl(urlA);
+    const parsedUrlB = Util.parseUrl(urlB);
+
+    if (!parsedUrlA || !parsedUrlB) {
```

Comment:
this one doesnt work with relative URLs 🤔 

I'd expect these tests to pass 
```
    it("returns true for same relative paths", function() {
      const hasSameHost = RouterUtil.hasSameHost(
        "/some/path",
        "/some/path"
      );
      expect(hasSameHost).toEqual(true);
    });
    it("returns false for different relative paths", function() {
      const hasSameHost = RouterUtil.hasSameHost(
        "/some/path",
        "/some/other/path"
      );
      expect(hasSameHost).toEqual(false);
    });
    it("returns true for same relative and absolute paths", function() {
      const hasSameHost = RouterUtil.hasSameHost(
        CURRENTHOST+"/some/path",
        "/some/path"
      );
      expect(hasSameHost).toEqual(true);
    });
    it("returns true for same relative and absolute paths", function() {
      const hasSameHost = RouterUtil.hasSameHost(
        CURRENTHOST+"/some/path",
        "/some/other/path"
      );
      expect(hasSameHost).toEqual(false);
    });
```
(you have to mock CURRENTHOST)

Reply:
I see what you're saying for relative URLs. Since this compares the host, any relative URL should probably return true as they have the same host?

Reply:
Now, for relative url it will attach the local origin and compare the local host